tests/Runtimes/RuntimesTest.php

```diff
@@ -96,6 +96,16 @@ public function setUp(): void
                 'code' => $functionsDir . '/dotnet-5.0.tar.gz',
                 'command' => 'dotnet dotnet.dll',
                 'timeout' => 15,
+            ],
+            'java-11' => [
+                'code' => $functionsDir . '/java-11.tar.gz',
+                'command' => 'java HelloWorld',
+                'timeout' => 15,
+            ],
+            'kotlin-jar' => [
```

Comment:
Right now this is not run.

I'll think of something so we can allow running multiple scripts on the same runtime.

Reply:
ok.

Reply:
may be restructure the array to specify the runtime to use to execute the test

Reply:
Will do this once yours is merged, don't wanna create conflicts 👍🏻